src/functionBased/components/InputTodo.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react'
+import { FaPlusCircle } from 'react-icons/fa'
+
+
+const InputTodo = props => {
+	const [inputText, setInputText] = useState({
+		title: ''
+	})
+
+	const onChange = e => {
+		setInputText({
+			...inputText,
+			[e.target.name]: e.target.value,
+		})
+	}
+
+	const handleSubmit = e => {
+		e.preventDefault()
+		if (inputText.title.trim()) {
+			props.addTodoProps(inputText.title)
+			setInputText({
+				title: ''
+			})
+		} else {
+			alert('The field cannot be empty')
+		}
+
+	}
+
+	return (
+		<form onSubmit={handleSubmit} className="form-container">
+			<input 
+				type='text'
+				className='input-text'
+				placeholder='Add todo...'
+				value={inputText.title}
+				name='title'
+				onChange={onChange}
+			/>
+			<button className="input-submit">
+				<FaPlusCircle 
+					style={{
+						color: "darkcyan",
+						fontSize: "20px",
+						marginTop: "2px"
+					}}
```

Comment:
- [ ] Its good practice to separate our styling in a separate CSS file so as to have a cleaner-looking codebase. kindly do so.